docs/rules/unique-dependencies.md

```diff
@@ -5,3 +5,28 @@
 💡 This rule is manually fixable by [editor suggestions](https://eslint.org/docs/latest/use/core-concepts#rule-suggestions).
 
 <!-- end auto-generated rule header -->
+
+This rule checks that every dependency is just added once to a [`package.json` key specifying dependencies](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/blob/main/src/rules/unique-dependencies.ts#L8-L16).
+
+Example of **incorrect** code for this rule:
+
+```json
+{
+	"dependencies": {
```

Comment:
Looks like this file got some spaces instead of a tab:

![Screenshot of the rendered incorrect code output from the file, with "dependencies" indented too much](https://github.com/user-attachments/assets/15aab406-73d2-4cbf-bbdc-114528880ffe)


```suggestion
    "dependencies": {
```

Reply:
whoopsie! fixed in 1e41854b760707447183f5b3e1ca87ab3b7412af

good catch 👁️